spec/workers/nudge_candidates_worker_spec.rb

```diff
@@ -43,6 +50,17 @@
           I18n.t!('candidate_mailer.nudge_unsubmitted_with_incomplete_courses.subject'),
         )
       end
+
+      it 'sends email to candidates with incomplete personal statement on their application' do
+        described_class.new.perform
+
+        email = email_for_candidate(application_form_with_no_personal_statement.candidate)
+
+        expect(email).to be_present
+        expect(email.subject).to include(
+          I18n.t!('candidate_mailer.nudge_unsubmitted_with_incomplete_personal_statement.subject'),
```

Comment:
`t!` is the same as `t` except it raises an exception for missing translations. TBH though I only used it to be consistent with the methods above.

Reply:
The translate with `!` is a wrapper for translate that adds `:raise => true` and if no translation is found, it will raise `I18n::MissingTranslationData`.

Reply:
ah, makes sense, maybe I should be using the `!` more often 😄